lib/tee/forwarder.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Membrane.Element.Tee.Filter do
+  use Membrane.Element.Base.Filter
+
+  def_input_pad :input,
+    availability: :always,
+    mode: :pull,
+    demand_unit: :buffers,
+    caps: :any
+
+  def_output_pad :output_static,
+    availability: :always,
+    mode: :pull,
+    caps: :any
+
+  def_output_pad :output_dynamic,
+    availability: :on_request,
+    mode: :push,
+    caps: :any
+
+  defp pass_to_all_outputs(pads, action, value, state) do
```

Comment:
We usually keep private function declarations after functions they were used in or after public function declarations.